pkg/api/domain_test.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import (
+	"testing"
+)
+
+func TestDomainForService(t *testing.T) {
```

Comment:
I know I am paranoid :)

Reply:
I do this too! Mostly so that when someone needs to extend the trivial method or adds a new one, they find an existing file with existing tests. I found people much likely to add new tests that way.